streams/src/test/java/org/apache/kafka/streams/processor/internals/ProcessorTopologyTest.java

```diff
@@ -1264,6 +1281,31 @@ public void process(final Record<String, String> record) {
         }
     }
 
+    private static class StatefulProcessorWithInitialization implements Processor<String, String, Void, Void> {
+        private KeyValueStore<String, String> store;
+        private final String storeName;
+        private final String initialKey;
+        private final String initialValue;
+
+        public StatefulProcessorWithInitialization(final String storeName, final String initialKey, final String initialValue) {
+            this.storeName = storeName;
+            this.initialKey = initialKey;
+            this.initialValue = initialValue;
+        }
+
+        @Override
+        public void init(final ProcessorContext<Void, Void> context) {
+            store = context.getStateStore(storeName);
+            store.put(initialKey, initialValue);
+        }
+
+        @Override
+        public void process(final Record<String, String> record) {
+            store.put(record.key(), record.value());
```

Comment:
nit: seems we can leave `process()` empty, as we never pipe any data through this processor?